polaris-react/src/components/Inline/Inline.scss

```
@@ -0,0 +1,117 @@
+.Inline {
+  --pc-stack-spacing: var(--p-space-4);
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+  margin-left: calc(-1 * var(--pc-stack-spacing));
+
+  > .Item {
+    margin-left: var(--pc-stack-spacing);
+    max-width: 100%;
+  }
+}
+
+.noWrap {
+  flex-wrap: nowrap;
+}
+
+.spacing0 {
+  --pc-stack-spacing: var(--p-space-0);
+}
+
+.spacing025 {
+  --pc-stack-spacing: var(--p-space-025);
+}
+
+.spacing05 {
+  --pc-stack-spacing: var(--p-space-05);
+}
+
+.spacing1 {
+  --pc-stack-spacing: var(--p-space1);
```

You'll want to change this
```suggestion
  --pc-stack-spacing: var(--p-space-1);
```